app/models/Claim.scala

```diff
@@ -28,13 +28,20 @@ sealed trait Claim extends DateFormatters {
   val serviceType: ServiceType
   val claimStartDate: LocalDate
   val lrn: Option[String]
-  def url(searched: Boolean): String = controllers.routes.ClaimDetailController.claimDetail(caseNumber, serviceType, searched).url
+  val claimStatus: ClaimStatus
+  def url(searched: Boolean): String = claimStatus match {
+    case InProgress => controllers.routes.ClaimDetailController.inProgressClaimDetail(caseNumber, serviceType, searched).url
+    case Closed => controllers.routes.ClaimDetailController.inProgressClaimDetail(caseNumber, serviceType, searched).url
```

Comment:
Should this not be controllers.routes.ClaimDetailController.closedClaimDetail

Reply:
yes, good spot